File: Bab 8/8.6.2.2 - 1 Count Vocal Character in String.java
import java.util.Scanner;

public class Main{

	public static void main(String[] args) {
		
		String string = "";
		int penghitung = 0;
		int jumlah = 0;

		Scanner sc = new Scanner(System.in);

		System.out.println("Masukan angka:\n");

		try{

			string = sc.next();

		}catch(Exception e){

		}

		jumlah = 0;

		for (penghitung = 0; penghitung < string.length(); penghitung++)
		{
			if ((string.chartAt(penghitung) == 'a') || (string.chartAt(penghitung) == 'i') || (string.chartAt(penghitung) == 'u') || (string.chartAt(penghitung) == 'e') || (string.chartAt(penghitung) == 'o'))
			{
				jumlah++;
			}
		}

		System.out.println(jumlah);
	}
}

File: Bab 8/8.6.2.2 - 1 Count Vocal Character in String.c
#include <stdio.h>
#include <string.h>

int main()
{
	
	char string[50];
	int penghitung, jumlah;

	printf("Masukan kata:\n");
	scanf("%s", &string);
	printf("\n");

	jumlah = 0;

	for (penghitung = 0; penghitung < strlen(string); penghitung++)
	{
		if ((string[penghitung] == 'a') || (string[penghitung] == 'i') || (string[penghitung] == 'u') || (string[penghitung] == 'e') || (string[penghitung] == 'o'))
		{
			jumlah++;
		}
	}

	printf("%d\n", jumlah);

	return 0;
}